lib/main_common.pm

```diff
@@ -1164,7 +1164,7 @@ sub load_consoletests {
         loadtest "console/zypper_ref";
     }
     loadtest "console/ncurses";
-    loadtest "console/yast2_lan" unless is_bridged_networking;
+    loadtest "console/yast2_lan" if (!is_bridged_networking && is_sle('<15-SP2'));
```

Comment:
As we have discussed previously, I don't think that this is a good solution, as once we have SP3 we will have to remember to change it. As an alternative we have thought of `EXCLUDE_MODULES` setting in the Functional job group.

Reply:
oh! you're right, thanks, I forgot for a moment that they also have yaml there!